src/Network.cpp

```diff
@@ -860,9 +861,40 @@ Network::Netresult Network::get_scored_moves(
     if (ensemble == DIRECT) {
         assert(rotation >= 0 && rotation <= 7);
         result = get_scored_moves_internal(state, planes, rotation);
+    } else if (ensemble == MULTI_AVG && rotation > 1) {
+        // MULTI_AVG rotation is the # of random rotations to provide
+        assert(rotation >= 1 && rotation <= 8);
+        std::vector<int> r_list{0, 1, 2, 3, 4, 5, 6, 7};
+        // Need to account for extra 2 in m_squaresize.
+        float policy[(BOARD_SIZE + 2) * (BOARD_SIZE + 1)] = {0};
```

Comment:
Use a std::vector or (at worst) std::array. There should already be constants for the policy array size.

Reply:
The closest constant I've found so far from FastBoard.h is `MAXSQ = ((BOARD_SIZE + 2) * (BOARD_SIZE + 2))`. I'll keep looking.

In get_scored_moves_internal I see `std::vector<float> policy_data(OUTPUTS_POLICY * width * height)` which is much larger since OUTPUTS_POLICY is 2

Reply:
Strange how the simple change to std::vector has broken this in confusing ways. Will try to clean it up, sorry about pushing a commit with non-working code. My lack of C++ experience is showing.